latest/src/app/documentation/demos/datagrid/datagrid.demo.html

```diff
@@ -23,5 +23,21 @@ <h4>Omit "select-all" in datagrid header</h4>
         </clr-datagrid>
 
         <clr-code-snippet [clrCode]="noSelectAllExample"></clr-code-snippet>
+
+        <h3>Workaround for Datagrid issues</h3>
```

Comment:
We don't want to put workarounds for clarity bugs inside clarity-addons documentation, because we then have to monitor these bugs and remove the workarounds afterwards.
Normal procedure should be to search the bug in the open clarity issues (where the workaround for this bug is already posted).